wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -639,6 +643,7 @@ func RunCommand(ctx context.Context, pk *scpacket.FeCommandPacketType) (scbus.Up
 	}
 	runPacket.Command = strings.TrimSpace(cmdStr)
 	runPacket.ReturnState = resolveBool(pk.Kwargs["rtnstate"], isRtnStateCmd)
+	runPacket.IsSudo = sudoArg || IsSudoCommand(cmdStr)
```

Comment:
slightly different logic here. `if _, ok := pk.Kwargs[KwArgSudo]; ok` then use sudoArg otherwise use IsSudoCommand.  that lets the user use `[sudo=0]` to force IsSudo to be false.

Reply:
fixed